lib/mayaUsd/fileio/utils/meshReadUtils.cpp

```diff
@@ -566,7 +566,7 @@ namespace
 
 // This can be customized for specific pipelines.
 bool
-UsdMayaMeshUtil::GetEmitNormalsTag(const MFnMesh& mesh, bool* value)
+UsdMayaMeshUtil::getEmitNormalsTag(const MFnMesh& mesh, bool* value)
```

Comment:
We should probably also change the namespace from `UsdMayaMeshUtil` to `UsdMayaMeshReadUtils`?

Reply:
see 1e492f25f45430e9b5d94c846baf5fc444e37519